frontend/src/components/editor/ai/ai-completion-editor.tsx

```diff
@@ -178,6 +192,8 @@ export const AiCompletionEditor: React.FC<Props> = ({
 
   const showCompletionBanner =
     enabled && triggerImmediately && (completion || isLoading);
+  // Set default output area to below if not specified
+  outputArea = outputArea === undefined ? "below" : outputArea;
```

Comment:
nit, but cleaner: `outputArea = outputArea ?? "below"`

just for fun, you could also do this but i don't find it easy to read:

```
`outputArea ??= "below"`
```